invenio_cli/helpers/commands.py

```diff
@@ -228,8 +228,16 @@ def demo(self):
         command = ['pipenv', 'run', 'invenio', 'rdm-records', 'demo']
         subprocess.run(command, check=True)
 
-    def run(self):
+    def run(self, stop):
         """Run development server and celery queue."""
+        if stop:
+            command = [
```

Comment:
There is a `stop_containers` function in the `docker_helper` file. It might be good using that :)

Reply:
Will do!